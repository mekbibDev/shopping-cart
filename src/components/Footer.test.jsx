import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import Footer from "./Footer";

describe("Footer", () => {
  it("matches the taken snapshot", () => {
    const {container} = render(<Footer></Footer>)
    expect(container).toMatchSnapshot()
  })
})