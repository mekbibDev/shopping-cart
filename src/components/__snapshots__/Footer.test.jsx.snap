// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > matches the taken snapshot 1`] = `
<div>
  <footer>
    <address>
      <h1>
        Contact us
      </h1>
      Have a question or need assistance? We're here to help! Feel free to reach out to us using the contact information below:
      <h2>
        Customer Support
      </h2>
      <nav>
        <a
          href="+1-800-123-4567"
        >
          1-800-123-4567
        </a>
        <a
          href="mailto:support@swiftcartexpress.com"
        >
          support@swiftcartexpress.com
        </a>
      </nav>
      <h2>
        Business enquires
      </h2>
      <nav>
        <a
          href="+1-800-765-4321"
        >
          1-800-765-4321
        </a>
        <a
          href="mailto:business@swiftcartexpress.com"
        >
          business@swiftcartexpress.com
        </a>
      </nav>
      <h2>
        Address
      </h2>
      SwiftCart Provisions Express 123 Shopping Street Cityville, State 54321 Country
    </address>
  </footer>
</div>
`;
